pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -283,21 +294,20 @@ def test_loc_getitem_duplicates_multiindex_missing_indexers(indexer, is_level1,
 
     if indexer == []:
         expected = s.iloc[[]]
-    elif is_level1:
+    elif level == 1:
         expected = Series([0, 3, 6], index=MultiIndex.from_product(
             [['A', 'B', 'C'], ['foo']], names=['one', 'two'])).sort_index()
+    elif level == -1:
+        # Empty: use idx[:0] as index to have appropriate (hidden) labels
+        expected = Series([], index=idx[:0], dtype='int64')
```

Comment:
i think this is the same as the `expected = s.iloc[[]]` above.

with the exception case now removed, I think that this whole test could be simplified..

```python
@pytest.mark.parametrize('indexer, expected', [
    ([], []),  # empty ok
    (['A'], slice(3)),
    (['A', 'D'], slice(3)),
    (['D', 'E'], []),  # no values found - fine
    (['D'], []),  # same, with single item list: GH 27148
    (pd.IndexSlice[:, ['foo']], slice(2, None, 3)),
    (pd.IndexSlice[:, ['foo', 'bah']], slice(2, None, 3))
])
def test_loc_getitem_duplicates_multiindex_missing_indexers(indexer, expected):
    # GH 7866
    # multi-index slicing with missing indexers
    idx = MultiIndex.from_product([['A', 'B', 'C'],
                                   ['foo', 'bar', 'baz']],
                                  names=['one', 'two'])
    s = Series(np.arange(9, dtype='int64'), index=idx).sort_index()
    expected = s.iloc[expected]
    result = s.loc[indexer]
    tm.assert_series_equal(result, expected)
```

Reply:
Right, thanks! I only changed the name of the second parameter to "pos".